tutorials/publish_subscribe/cs/README.md

```diff
@@ -0,0 +1,86 @@
+# Tutorial: Publish-Subscribe
+
+This code is part of the Connext Publish-Subscribe tutorial.
+
+## Building the Example :wrench:
+
+You can build the example following the instructions in the tutorial, or you can
+build it as follows.
+
+1. Setting up the environment for Connext:
```

Comment:
It was just to have rtiddsgen in the PATH, but it is true that it is a lot of information just to call rtiddsgen instead of `<install dir>/bin/rtiddsgen`